### PR TITLE
Feature manage bookings integration #31

### PR DESCRIPTION
**Pull request by seblaifa3315**
### What this PR does (required):
Integrated management of bookings with requests so that it has the ability to:
- accept/decline a booking: 
- if the request hasn't bee accepted or declined the status is "PENDING" and the sitter has the option to accept or decline it by clicking directly on the status
- if the request has been accepted, the sitter still has the option to decline it by clicking directly on the status
- if the request has been declined, the sitter won't be able to accept the request anymore
- Also, the status of the pas booking can't be changed

### Screenshots / Videos (front-end only):
manageBookingsIntegration

### Any information needed to test this feature (required):
To test the feature, please follow those steps:
- first login with email: "johndoe@gmail.com" and password: "johndoe"
- Then, once you are logged in, go to the path "/manageBookings"

### Any issues with the current functionality (optional):
- I added the status (pending/approved/declined) the the next booking because I thought that at this point the sitter could still manage this booking.
- I haven't been able to reproduced the calendar according to the mockup yet... I will continue to work on it and create a new PR.

### CODE REVIEW

File: client/src/pages/ManageBookings/ManageBookings.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    avatar?: string | undefined,
+  ): void => {
+    changeStatus(accepted, declined, requestId, sitterId, avatar).then((res) => {
+      const index = sitterRequests.findIndex((request: RequestApiData) => {
+        return request._id === requestId;
+      });
+      if (index !== -1) {
+        const newSitterRequests = [...sitterRequests];
+        newSitterRequests[index] = res;
+        console.log(res);
```
Comment (ezeilo-su) [inline client/src/pages/ManageBookings/ManageBookings.tsx:36]: Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!

**Review by ezeilo-su — COMMENTED** [review]
Good progress with this one.

**Comment by rajivtitus** [comment]
We can have another look at this once we've completed the minor changes within the Bookings UI and request controllers branches 👍🏽

**Comment by rajivtitus** [comment]
@seblaifa3315 Since the previous bookings PRs are approved, let's update this PR with the new changes (once they've been done in those PRs) 👍🏽

**Review by rajivtitus — APPROVED** [review]
I didn't find any file changes pertaining to the Bookings Integration ticket, the rest of the changes look good (maybe we can update the title and template of this PR accordingly)

**Pull request merged**